judgments/tests/factories.py

```diff
@@ -0,0 +1,52 @@
+from typing import Any
+from unittest.mock import Mock
+
+from caselawclient.models.judgments import Judgment
+
+
+class JudgmentFactory:
+    # "name_of_attribute": ("name of incoming param", "default value")
+    PARAMS_MAP: dict[str, tuple[str, Any]] = {
+        "uri": ("uri", "test/2023/123"),
+        "name": ("name", "Judgment v Judgement"),
+        "neutral_citation": ("neutral_citation", "[2023] Test 123"),
+        "court": ("court", "Court of Testing"),
+        "judgment_date_as_string": ("judgment_date_as_string", "2023-02-03"),
+        "is_published": ("is_published", False),
+        "is_sensitive": ("is_sensitive", False),
+        "is_anonymised": ("is_anonymised", False),
+        "has_supplementary_materials": ("has_supplementary_materials", False),
+        "is_failure": ("is_failure", False),
+        "source_name": ("source_name", "Example Uploader"),
+        "source_email": ("source_email", "uploader@example.com"),
+        "consignment_reference": ("consignment_reference", "TDR-12345"),
+        "assigned_to": ("assigned_to", ""),
+        "versions": ("versions", []),
+    }
+
+    @classmethod
+    def build(cls, **kwargs) -> Judgment:
+        judgment_mock = Mock(spec=Judgment, autospec=True)
+
+        if "html" in kwargs:
+            judgment_mock.return_value.content_as_html.return_value = kwargs.pop("html")
```

Comment:
Instead of the if else block, can we pass a second, `default`, argument to the pop method?

like so:

`judgment_mock.return_value.content_as_html.return_value = kwargs.pop("html", "<p>This is a judgment in HTML.</p>")`